src/tests/suite/daos_test.c

```diff
@@ -193,6 +201,7 @@ main(int argc, char **argv)
 		{"pool",	no_argument,		NULL,	'p'},
 		{"cont",	no_argument,		NULL,	'c'},
 		{"capa",	no_argument,		NULL,	'C'},
+		{"DTX",		no_argument,		NULL,	'X'},
```

Comment:
minor nit...make the option dtx rather than all caps

Reply:
Sure. I will change it.